File: C/Easy/3190.c
int minimumOperations(int *nums, int numsSize)
{
    int count = 0;
    int remainder;

    for (int i = 0; i < numsSize; i++)
    {
        remainder = nums[i] % 3;
        if (remainder != 0)
        {
            count += 1; // always 1 away from being divisible by 3
        }
    }

    return count;
}

File: Java/Easy/3190.java
class Solution {
    public int minimumOperations(int[] nums) {
        int count = 0;
        for (int i : nums) {
            if (i % 3 < (3 - (i % 3))) {
                count += i % 3;
            } else {

                count += (3 - (i % 3));
            }
        }

        return count;
    }
}